fdm-app/app/components/custom/soil/form.tsx

```
@@ -40,27 +40,29 @@ export function SoilAnalysisForm(props: {
     soilParameterDescription: SoilParameterDescription
     action: string
 }) {
-    const { soilAnalysis, soilParameterDescription, action } = props
+    const { soilAnalysis, soilParameterDescription } = props
+
+    const defaultValues: { [key: string]: any } = {}
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)